index.js

```diff
@@ -8,33 +8,40 @@ require('dotenv').config();
 const bot = new Telegraf(process.env.BOT_TOKEN);
 
 routes.forEach(item => {
-  bot.hears(item.firstMatch, (ctx) => {
-    const message = ctx.match.input;
-    if (!item.secondMatch) {
-      return item.action()
-        .then(res => ctx.reply(res))
-        .catch(() => ctx.reply('maaf, ada kesalahan di server hayati kak'));
-    }
+    bot.hears(item.firstMatch, (ctx) => {
+        const message = ctx.match.input;
+        if (!item.secondMatch) {
+            return item.action()
+                .then(res => ctx.reply(res))
+                .catch(() => ctx.reply('maaf, ada kesalahan di server hayati kak'));
+        }
 
-    if (message.match(item.secondMatch)) {
-      return item.action()
-        .then(res => ctx.reply(res))
-        .catch(() => ctx.reply('maaf, ada kesalahan di server hayati kak'));
-    } else {
-      return ctx.reply(utils.getRandomMessage());
-    }
-  });
+        if (message.match(item.secondMatch)) {
+            return item.action()
+                .then(res => ctx.reply(res))
+                .catch(() => ctx.reply('maaf, ada kesalahan di server hayati kak'));
+        } else {
+            return ctx.reply(utils.getRandomMessage());
+        }
+    });
 });
 
 bot.hears(/((hayati jahat)|(kamu jahat))/, ctx => {
-  ctx.replyWithAudio({
-    source: fs.createReadStream(path.join(__dirname, 'assets/audio/zainudin-kejam.mp3')),
-    title: 'Zainudin 😭'
-  });
+    ctx.replyWithAudio({
+        source: fs.createReadStream(path.join(__dirname, 'assets/audio/zainudin-kejam.mp3')),
+        title: 'Zainudin 😭'
+    });
+});
+
+bot.hears(/((pada ngopi)|(kopi hitam))/, ctx => {
```

Comment:
please change it with static string
```javascript
bot.hears('udah pada ngopi?')
```
if you just want to learn and figuring out